### ISSUE TITLE
lncli listpayments always reports fee as 0.

### ISSUE THREAD

**Issue by vegardengen** [lightningnetwork/lnd#801]
Is there a reason for that? The below small patch gave me the fee (in millisatoshis)

```
vegard@engen-btc:~/gocode/src/github.com/lightningnetwork/lnd$` git diff
diff --git a/rpcserver.go b/rpcserver.go
index 13cf6f5..ededa3c 100644
--- a/rpcserver.go
+++ b/rpcserver.go
@@ -2927,6 +2927,7 @@ func (r *rpcServer) ListPayments(ctx context.Context,
                        Value:           int64(payment.Terms.Value.ToSatoshis()),
                        CreationDate:    payment.CreationDate.Unix(),
                        Path:            path,
+                       Fee:             int64(payment.Fee),
                        PaymentPreimage: hex.EncodeToString(payment.PaymentPreimage[:]),
                }
        }
```

**Comment by Roasbeef** [comment]
Just looks like a cosmetic bug, we forgot to assign the fee. Can you make a PR for that? Thanks!

**Comment by vegardengen** [comment]
PR created.

**Comment by Roasbeef** [comment]
Fixed by #804.
